src/main_macro.rs

```diff
@@ -29,23 +31,68 @@
 ///     println!("{}", x);
 /// });
 /// ```
+///
+/// With command line arguments.
+///
+/// ```rust,ignore
+/// #[macro_use] extern crate quicli;
+/// use quicli::prelude::*;
+/// use std::path::PathBuf;
+///
+/// #[derive(Debug, StructOpt)]
+/// struct Cli {
+///     /// The config file for the cli app.
+///     #[structopt(long = "config", short = "c", parse(from_os_str))]
+///     config: PathBuf,
+/// }
+///
+/// main!(|args: Cli| {
+///     let x = read_file(&args.config)?;
+///     println!("{}", x);
+/// });
+/// ```
+///
+/// With command line arguments including a verbosity flag.
+///
+/// ```rust,ignore
```

Comment:
why ignore and not no_run?

Reply:
With no_run the tests fail. I think custom derive doesn't work in doctests.